### ISSUE TITLE
Initial Audio Neural Net

### ISSUE THREAD

**Issue by Olivia-Roscoe** [alewan/capstone#24]
**Obj:**  Write an initial CNN to train on the audio spectral images

**AC:**
1. Output - final test accuracy
2. Input - image to be classified
3. Checkpoint of trained net
4. Built using pytorch
